Representatives.md

```diff
@@ -0,0 +1,74 @@
+The jQuery Foundation invests significant resources in its efforts to drive Web standards on behalf of developers. The Standards Representatives coordinates those efforts, communicating on a regular basis with both the community and the projects hosted within the jQuery Foundation.
+
+The Standards Representatives will:
+
+- Know what Foundation projects are doing related to standards, and what departments or projects are involved to properly disseminate information and solicit feedback.
+- Act as ambassadors to the wider web development community, and provide feedback to the Foundation regarding its impact in that community.
+- State their affiliation with the jQuery Foundation, particularly in conferences or meetings where their attendance is a result of affiliation with, or funded by, the Foundation.
+- Provide a high-level summary of the meetings they attend, and distribute those reports as widely as possible. If there is no confidential information, public distribution is preferred.
+- Distribute information about upcoming standards meetings within the jQuery Foundation to ensure that they are attended by the appropriate representatives.
+- Inform the jQuery Foundation and interested projects of upcoming votes in the standards bodies, and take feedback on the appropriate way to vote on the matter.
+- Regularly provide reasonable data for expenses reimbursement regarding any available budget to attend meetings or events associated to their respective representation work.
+
+_Contributions can be found at [this document](https://github.com/jquery-foundation/standards/blob/master/Contributions.md)._
+
+## Standards team coordinator
+
+- Dave Methvin, @dmethvin
+
+## Advisory Committee Rep
+
+- Kris Borchers: @kborchers
+
+## Audio WG
+
+- Alexander Schmitz: @arschmitz
+
+## Browser Extension Community Group
+
+- Brian Kardell: @bkardell
+
+## Cascading Style Sheets (CSS) WG
+
+- Brian Kardell: @bkardell
+
+## Device and Sensors WG
+
+- Rick Waldron: @rwaldron
```

Comment:
Technically I am, though I haven't been super active there and the things I've done there I have mostly explicitly taken off the jQuery hat because I didn't feel like I was in a position on some of those things to say too much officially.  I'd like to more though moving forward, especially if/as we launch more efforts - so I'd like to keep it.  

Yeah, some of the CG stuff/Task force stuff is tricky.... For example, Extensible Web Community Group was affiliated with my personal W3C account and signed off by my employer, so technically I represent no one there officially.